src/GraphQL.DataLoader.Tests/DataLoaderQueryTests.cs

```diff
@@ -377,7 +377,7 @@ public void EnumerableDataLoaderResult_Works()
         /// <summary>
         /// Exercises Execution.ExecutionNode.ResolvedType for children of children, verifying that
         /// <see cref="Execution.ExecutionNode.GraphType"/> is returning proper values. Without a dataloader,
-        /// <see cref="Execution.ExecutionStrategy.SetArrayItemNodes(Execution.ExecutionContext, Execution.ArrayExecutionNode)"/>
+        /// <see cref="Execution.ExecutionStrategy.CompleteNode(Execution.ExecutionContext, Execution.ExecutionNode)"/>
```

Comment:
@Shane32 I am not sure if the comment is correct.

Reply:
The original comment was correct.  See https://github.com/graphql-dotnet/graphql-dotnet/blob/c356fda817bebe7267489be44a17b832a240f409/src/GraphQL/Execution/ExecutionStrategy.cs#L154

See #2064 

```suggestion
        /// <see cref="Execution.ExecutionStrategy.SetArrayItemNodes(Execution.ExecutionContext, Execution.ArrayExecutionNode)"/>
```